docs/sources/flow/reference/components/loki.relabel.md

```diff
@@ -0,0 +1,182 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/loki.relabel
+title: loki.relabel
+---
+
+# loki.relabel
+
+The `loki.relabel` component rewrites the label set of each log entry passed to
+its receiver by applying one or more relabeling `rule`s and forwards the
+results to the list of receivers in the component's arguments.
+
+<!-- TODO(@tpaschalis) Add note about loki.transform
+  To manipulate the log entry itself, you can look at the loki.transform
+  component which allows you to run one or more 'stages' on the log entries.
+ -->
+
+If no labels remain after the relabeling rules are applied, then the log entry
+are dropped.
+
+The most common use of `loki.relabel` is to filter log entries or standardize
+the label set that is passed to one or more downstream receivers. The `rule`
+blocks are applied to the label set of each log entry in order of their
+appearance in the configuration file.
+
+Multiple `loki.relabel` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+loki.relabel "LABEL" {
+  forward_to = RECEIVER_LIST
+
+  rule {
+    ...
+  }
+
+  ...
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`forward_to` | `list(receiver)` | Where the log entries should be forwarded to, after relabeling takes place. | | yes
```

Comment:
You might be able to make this a bit shorter, WDYT? 

```suggestion
`forward_to` | `list(receiver)` | Where to forward log entries after relabeling. | | yes
```

Reply:
Nice edit, Robert!   I'm not needed anymore.  :-(